libpod/container.go

```diff
@@ -374,8 +376,32 @@ func (c *Container) Exec(cmd []string, tty bool, stdin bool) (string, error) {
 
 // Attach attaches to a container
 // Returns fully qualified URL of streaming server for the container
-func (c *Container) Attach(stdin, tty bool) (string, error) {
-	return "", ErrNotImplemented
+func (c *Container) Attach(noStdin bool, keys string) error {
```

Comment:
Can we pass in the streams for stdin/out/err instead of assuming they're os.Stdin/out/err? Just add them as arguments?